src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -21,16 +21,16 @@ import {ShadowsocksSessionConfig} from '../tunnel';
 // DON'T use these methods outside of this folder!
 
 // Parses an access key string into a ShadowsocksConfig object.
-export function accessKeyToShadowsocksSessionConfig(accessKey: string): ShadowsocksSessionConfig {
+export function staticKeyToShadowsocksSessionConfig(staticKey: string): ShadowsocksSessionConfig {
```

Comment:
Perhaps we should do accessKey -> ServiceConfig, which is id + name + sessionConfig(static or dynamic)

Then have a sessionConfig ->  AccessKey for serialization

Reply:
I'm not exactly sure what you're proposing here, we can sync offline